packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -139,6 +139,13 @@ const Home = ({value}: Props) => {
                             <HelloJS />
                             &nbsp;
                             <HelloTS message="it works!" />
+                            <br />
+                            <br />
+                            Client-side JS works if this counter increments: {counter}
+                            <br />
+                            <br />
+                            App Extensions work if you can navigate to this link:{' '}
+                            <Link to="/sample-page">sample page</Link>.
```

Comment:
Are you planning on leaving this code in here? I wonder in the distant future we add an API method to allow the end user to get all the extensions so they can be used in situations like this where you want to list them out dynamically instead of having static links.

Reply:
I removed the code. It was only there to facilitate testing. I believe it would be helpful to have a method for listing all the extensions.